test/Riok.Mapperly.IntegrationTests/Mapper/TestMapperWithStaticMethods.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using Riok.Mapperly.Abstractions;
+using Riok.Mapperly.IntegrationTests.Dto;
+using Riok.Mapperly.IntegrationTests.Models;
+
+namespace Riok.Mapperly.IntegrationTests.Mapper
+{
+    [Mapper]
+    public partial class TestMapperWithStaticMethods : ITestStaticInterface
+    {
+        public static partial double DirectDouble(double value);
+
+        public static partial int StaticAbstractDirectInt(int value);
+
+        [MapperIgnoreTarget(nameof(TestObjectDto.IgnoredStringValue))]
+        [MapperIgnoreTarget(nameof(TestObjectDto.IgnoredIntValue))]
+        [MapperIgnoreSource(nameof(TestObject.IgnoredIntValue))]
+        [MapProperty(nameof(TestObject.RenamedStringValue), nameof(TestObjectDto.RenamedStringValue2))]
+        [MapProperty(
+            new[] { nameof(TestObject.UnflatteningIdValue) },
+            new[] { nameof(TestObjectDto.Unflattening), nameof(TestObjectDto.Unflattening.IdValue) }
+        )]
+        [MapProperty(
+            nameof(TestObject.NullableUnflatteningIdValue),
+            nameof(TestObjectDto.NullableUnflattening) + "." + nameof(TestObjectDto.NullableUnflattening.IdValue)
+        )]
+        [MapperIgnoreObsoleteMembers]
+        private static partial TestObjectDto MapToDtoInternal(TestObject testObject);
+
+        public static partial IEnumerable<TestObjectDto> MapAllDtos(IEnumerable<TestObject> objects);
+
+        public static TestObjectDto MapToDto(TestObject src)
+        {
+            var target = MapToDtoInternal(src);
+            target.StringValue += "+after-map";
+            return target;
+        }
```

Comment:
I think this test case can be removed as it is unit tested and it also covered in other integration tests. Sorry for the misunderstanding.

Reply:
Just saw the older, still open, discussion regarding this.